static/styles/portico/portico_signin.css

```diff
@@ -927,6 +927,20 @@ button#register_auth_button_gitlab {
     font-size: 22px;
 }
 
+#clint_type_section {
+    position: relative;
+    top: 15px;
+    margin-bottom: 20px;
+    font-size: 22px;
+}
+
+#authorization_grant_type_section {
```

Comment:
Merge these two selectors:

```css
#id-1, #id-2 {
    /* rules */
}
```

Better yet, give both of them a class like `oauth_section`.